src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -629,7 +621,7 @@ export const WithdrawForm = () => {
           onSelect={onSelectNetwork}
         />
       </div>
-      {isInvalidNobleAddress && (
+      {toAddress && Boolean(exchange) && !isValidDestinationAddress && (
```

Comment:
is requiring exchange correct here? you could in theory just send to noble if you wanted to right? perhaps this should be exchange or manually set noble chain id?

Reply:
i dont think this error message here is really necessary tbh because there's already an additional `isValidDestinationAddress` error message, so if you're sending to noble without coinbase then it will still error there, but kept this behavior same as before for now

Reply:
ah i see, we should probs just remove then - will only cause confusion in the future imo

Reply:
I think this error message was specific for exchange, so that the User knows that they need to use a noble address.